src/libs/decoders/stb_vorbis.h

```diff
@@ -1329,6 +1376,14 @@ static int getn(vorb *z, uint8 *data, int n)
       return 1;
    }
 
+   #ifdef __SDL_SOUND_INTERNAL__
```

Comment:
ifdefs to first column?

Reply:
STB indents preprocessors with the code level (atleast for the majority, he does have a few inconsistencies).  In this case, I want to match STB's style instead of DOSBox's.  If we were forking the STB files and taking them in our own direction, then we could make them 100% DOSBox' compliant. (but instead, we'll always be taking new STB files from upstream, and peppering these SDL changes into them, so prefer to follow his style).  Unfortunately with STB, there is no hope of up-streaming our changes, so we're stuck in this middle-ground.